deeprank/generate/GridTools.py

```diff
@@ -35,8 +35,8 @@ def __init__(self, molgrp,
             number_of_points(int, optional): number of points we want in
                 each direction of the grid.
             resolution(float, optional): distance(in Angs) between two points.
-            atomic_densities(dict, optional): dictionary of atom types with
-                their vdw radius, e.g. {'CA':1.7, 'C':1.7, 'N':1.55, 'O':1.52}
+            atomic_densities(dict, optional): dictionary of element types with
```

Comment:
atomType is the default name in the PDB file format no ? I have nothing against elementtype but I don't get why changing it

Reply:
The new code are calculating atomic densities based on element types (C,N,O,S) but not the previous atom types (e.g. CA, CB, C, N, O). Thus the new code will cover both backbone and sidechain atoms.  
Using element type is a convenient way to consider side chain atoms (which has ~ 40 atom types).

Reply:
sounds good